crates/database/src/states/reverts.rs

```diff
@@ -81,6 +82,39 @@ impl Reverts {
         state_reverts
     }
 
+    /// Compare two Reverts instances, ignoring the order of elements
+    pub fn content_eq(&self, other: &Self) -> bool {
+        if self.0.len() != other.0.len() {
+            return false;
+        }
+
+        for (self_transition, other_transition) in self.0.iter().zip(other.0.iter()) {
+            if self_transition.len() != other_transition.len() {
+                return false;
+            }
+
+            let mut self_map: HashMap<Address, &AccountRevert> = HashMap::new();
+            for (addr, account_revert) in self_transition {
+                self_map.insert(*addr, account_revert);
+            }
+
+            for (addr, account_revert) in other_transition {
+                match self_map.get(addr) {
+                    Some(self_account_revert) if *self_account_revert == account_revert => {
+                        self_map.remove(addr);
+                    }
+                    _ => return false,
+                }
+            }
+
+            if !self_map.is_empty() {
+                return false;
+            }
+        }
```

Comment:
This does not look like a correct solution, there can be multiple reverts of same account.

Something like this is more preferred even if it is less performant:

```
let mut self_transition = self_transition.clone()
self_transition.sort()
other_transition.sort();
...
return self_transition == other_transition

```

Reply:
i just update, thank you